EventSubscriber/BeforeMessageProcessingSubscriber.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Auxmoney\OpentracingAmqplibRabbitMqBundle\EventSubscriber;
+
+use Auxmoney\OpentracingAmqplibRabbitMqBundle\Consumer\Consumer;
+use Auxmoney\OpentracingBundle\Internal\Utility;
+use Auxmoney\OpentracingBundle\Service\Tracing;
+use OldSound\RabbitMqBundle\Event\AMQPEvent;
+use OldSound\RabbitMqBundle\Event\BeforeProcessingMessageEvent;
+use OpenTracing\Reference;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use const OpenTracing\Tags\SPAN_KIND;
+use const OpenTracing\Tags\SPAN_KIND_MESSAGE_BUS_CONSUMER;
+
+final class BeforeMessageProcessingSubscriber implements EventSubscriberInterface
+{
+    private const SPAN_NAME = 'Processing message from "%s" queue';
+    private const TAG_QUEUE_NAME = 'QueueName';
```

Comment:
We should follow the established naming conventions here, so I would suggest snake_case notation and a name like `message_bus.queue_name`.

Reply:
I like that!